pkg/agent/agentclient/client.go

```diff
@@ -201,11 +216,13 @@ func (a *AgentClient) remoteToProxy(conn net.Conn, connID int64) {
 
 	for {
 		n, err := conn.Read(buf[:])
+		klog.Infof("received %d bytes from proxy server", n)
```

Comment:
Might be worth switching some of these to verbose log levels? Eg. klog.V(4).Infof("received %d bytes from proxy server", n)

Reply:
created #28